samples/snippets/authorized_view_tutorial_test.py

```diff
@@ -19,6 +19,17 @@
 import pytest
 
 import authorized_view_tutorial  # type: ignore
+from conftest import prefixer  # type: ignore
+
+
+@pytest.fixture
+def table_id(
```

Comment:
We use the function name as a variable name below.

'table_id = f"...'
and
'yield table_id...`

is this intentional?
NOTE: the same issue applies to the function in the `view_test.py` file.